functions/assist-chatbox.js

```diff
@@ -10,3 +11,5 @@ app.http('assist_chatbox', {
   authLevel: 'anonymous',
   handler: (request) => chatBox(request, bot),
 });
+
+logger.info(`Assist Chatbox Function Started at ${new Date().toISOString()} on ${process.env.ENV}/${process.platform}.`, 'BOT-FUNC-STARTED');
```

Comment:
Why are we logging this? When the app starts successfully it means all functions are running. This feels redundant